src/components/main/Main.tsx

```diff
@@ -82,6 +84,7 @@ const Main = ({ children }: Props): JSX.Element => {
   );
   return (
     <GraaspMain
+      open={currentMember ? undefined : false}
```

Comment:
Why it is undefined instead of true? It is not the same to give false or undefined?

Reply:
As said in https://github.com/graasp/graasp-builder/pull/1434#discussion_r1754318118 I want to only set the prop when the user is **not logged in**, in the other cases I want to keep the default behaviour.